client/src/api/models/QueryObject.ts

```diff
@@ -1,7 +1,6 @@
-import { queryUrl } from "./QueryObjectUrl";
-import { ChoreWithAssignments } from "../../parent/models/ChoreWithAssignments";
+import { QueryObjectUrl } from "./QueryObjectUrl";
 
 export interface QueryObject {
```

Comment:
Könnte man aus dem QueryObject nicht auch noch einen Generic manchen?
Mit QueryObject<T> könnte man dann auch das _object_ Property vom Typ _T_ definieren statt als _any_.
Das könnte man ja vielleicht in einem späteren Schritt noch versuchen 🤓 .

Reply:
Ich hab 2 Tage lang versucht dieses noch tiefer hinunter zu geben.. ohne Erfolg. Deshalb der Wahnsinns comment beim pr ^^

Reply:
Ja, ich hab mir noch gedacht, dass da irgendwo ein Haken ist und ich mir das zu einfach vorstelle 😊.